projects/ion/src/lib/chip/chip.component.html

```diff
@@ -8,4 +8,10 @@
   [class]="'chip-' + size"
 >
   {{ label }}
+  <ng-container *ngIf="size === 'sm'">
+    <ion-icon color="#06439D" [type]="icon" size="16"></ion-icon>
+  </ng-container>
+  <ng-container *ngIf="size === 'md'">
+    <ion-icon color="#06439D" [type]="icon" size="20"></ion-icon>
+  </ng-container>
```

Comment:
I believe that exist a way of do this better.

Reply:
I made the style in scss. It's the alternative.

`.ion-btn-sm {
    ion-icon {
      ::ng-deep svg {
        height: 13px;
      }
   }
}`